lms/envs/common.py

```diff
@@ -2372,6 +2372,7 @@
 FINANCIAL_REPORTS = {
     'STORAGE_TYPE': 'localfs',
     'BUCKET': 'edx-financial-reports',
+    'CUSTOM_DOMAIN': 'edx-financial-reports.s3.amazonaws.com',
```

Comment:
I used `CUSTOM_DOMAIN` because this is what it is being named internally in `S3BotoStorage` and also is similar to existing name `AWS_S3_CUSTOM_DOMAIN`. 
I am not much familiar with S3 but it seems that `CUSTOM_DOMAIN` is used to point custom domain like `example.com` to S3 buckets, if that is the case than I think naming it `BUCKET_HOST` would be miss-leading.
But let me know if it seems better idea to change it I will update the PR.